### ISSUE TITLE
LICENSE and .gitignore files

### ISSUE THREAD

**Issue by tracker1** [spilled-ink/spilld#8]
Would suggest appropriate LICENSE and .gitignore files in the root of the project...

The LICENSE file would hold to the specific license version you are using... right now, there's not really clarity in this.

Also, may want to consider AGPL[1] license if you don't want your code used for creating a competing mail service around the software without contributing changes.  This would likely be more appropriate for your use case.

As to the .gitignore, mainly to prevent accidental checkins.

ASIDE: Glad to see this work... I've had a strong interest in email servers/services for some time.

[1] http://www.affero.org/oagpl.html

**Comment by chicks-net** [comment]
I agree, this looks like a cool project.  Have you tried it?  Did you find a better alternative?
